README.md

```diff
@@ -1 +1,2 @@
 # EasyApp
+# mohammed.taha test
```

Comment:
I would like to have a clean history of the merge and was planning to do rebase but this commit will appear in history with no need, so it's better if you can clean unneeded commit for clean history

Reply:
Also, the readme file as you can see was outdated while you edited it, this is why it's better to fetch the changes from the upstream before you committing your changes to make sure you have the latest before your commits.